### DISCUSSION TITLE
Why is DirectML NuGet so large?

### DISCUSSION THREAD

**Discussion by lostmsu** [amerkoleci/Vortice.Windows#454] [Q&A]
It includes binaries for every supported Windows platform, and each is >10MB. Isn't DirectML a component of OS? Why is the glue so large?

**Comment by amerkoleci** [comment]
Hi,
Probably because it includes the Debug version of every DLL and it has native DLL's for x64, x86, arm e arm64.
Probably the .Debug DLL's can be removed, what do you think?

**Reply by lostmsu** [reply]
I am not sure I understand. The other packages are under 1MB, and don't include any native bits at all. Why does DirectML require native DLLs in the package, debug or not?
